react/components/Upload/Link.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { ContentState } from 'draft-js'
+
+interface Props {
+  children: React.ElementType[]
+  contentState: ContentState
+  decoratedText: string
+  dir: string | null
+  entityKey: string
+  offsetKey: string
+}
+
+const Link = (props: Props) => {
+  const { contentState, entityKey, children } = props
+  const { url } = contentState.getEntity(entityKey).getData()
+
+  return (
+    <a href={url} className="blue">
```

Comment:
I don't have too much context, but why not use the render-runtime `Link` component?

Reply:
This file is unecessary. It was removed